docs/.vuepress/config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     title: 'Theme Packer',
     themeConfig: {
         repo: 'davidwarrington/theme-packer',
-        sidebar: ['/', '/guide/', '/configuration/', '/concepts/'],
+        sidebar: ['/', '/guide/', '/configuration/', '/concepts/','/recipes/'],
```

Comment:
Please could you add a space after the comma here? (Alternatively I think running `yarn lint --fix` should sort this.